cdis_oauth2client/blueprint.py

```diff
@@ -33,22 +33,20 @@ def get_authorization_url():
 @blueprint.route('/authorize', methods=['GET'])
 def do_authorize():
     """
-    Call ``authorize`` to authorize the user (authorize returns the username)
-    and store the username in the Flask session.
+    Call ``authorize`` to authorize the user by storing the access_token cookie
+    received from the OAuth2Client.
     """
-    flask.session['username'] = authorize(
-        current_app.oauth2,
-        current_app.config['USER_API'],
-        flask.request.args.get('code')
-    )
+    code = flask.request.args.get('code')
+    client = current_app.oauth2
+    flask.session['access_token'] = client.get_access_token(code)
```

Comment:
I think we don't want to use the session anymore?
We just want to save the session in cookie